src/styles/blocks/card.css

```diff
@@ -0,0 +1,13 @@
+.product-card {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
```

Comment:
do you really need this? all elements will be put in column direction by default